helm/prometheus-rules/templates/alerting-rules/mimir.rules.yml

```diff
@@ -9,18 +9,35 @@ spec:
   groups:
   - name: mimir
     rules:
+    # Coming from https://github.com/giantswarm/giantswarm/issues/30124
+    # This alert ensures Mimir containers are not restarting too often (flappiness).
+    # If it is not the the case, this can incur high costs by cloud providers (s3 api calls are quite expensive).
+    - alert: MimirRestartingTooOften
+      annotations:
+        description: '{{`Mimir containers are restarting too often.`}}'
+      expr: |
+        increase(
+          kube_pod_container_status_restarts_total{cluster_type="management_cluster", namespace="mimir"}[1h]
```

Comment:
I just executed that rules on `grizzly` and it will be paged because of prometheus-buddy restarting a lot...

Reply:
Then that's something we have to fix :)

Reply:
Can we scope this alert to pods requested S3 only ?

Reply:
I removed the buddy out of the equation and created a PM to investigate what is happening

Reply:
Considering this is a BH only alert (at least now), I would prefer that we keep it too big for now and reduce the scope later on but I'm open to suggestions from @hervenicol and @QuantumEnigmaa